resources/qml/TimelineView.qml

```diff
@@ -239,7 +239,7 @@ Item {
             Layout.rightMargin: Nheko.paddingLarge
 
             TextArea {
-                text: TimelineManager.escapeEmoji(preview.roomTopic)
+                text: preview.roomTopic ? TimelineManager.escapeEmoji(preview.roomTopic) : "This room is possibly inaccessible. If this room is private, you should remove it from the child list of this space."
```

Comment:
There are rooms, that simply don't have a topic :)

Reply:
Fixed